services/src/main/java/com/chatbot/services/AsyncServices/AsyncService.java

```diff
@@ -0,0 +1,22 @@
+package com.chatbot.services.AsyncServices;
+
+import java.io.IOException;
+
+import com.chatbot.services.protobuf.ChatServiceRequestOuterClass.ChatServiceRequest;
+import com.chatbot.services.protobuf.TriggerEventNotificationOuterClass.TriggerEventNotification;
+
+abstract class AsyncService {
+
+  protected static final String IMAGES_RECEIVED_MESSAGE = "The images have been received!";
+  protected static final String THANKS_FOR_ADDING_MESSAGE = "Thank You for Adding me";
+  protected static final String NOT_EXPECTING_IMAGE_MESSAGE =
+      "Sorry, we were not expecting any attachements from you.";
+
+  abstract void chatServiceRequestHandler(final ChatServiceRequest chatServiceRequest) throws Exception;
```

Comment:
Method descriptor comments.

Reply:
updated

Reply:
General convention is for writing this is

"Sends messages to a user using their userID "
No need to write "method to"

Can be an FYI for this PR